backend/config/settings/base.py

```diff
@@ -138,3 +138,7 @@
 # Available HTTP methods and headers that are allowed for the actual request.
 CORS_ALLOW_METHODS = list(default_methods) + []
 CORS_ALLOW_HEADERS = list(default_headers) + []
+
+# Email
+EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
+EMAIL_FILE_PATH = os.path.join(BASE_DIR, "emails")
```

Comment:
I guess, you should declare this at dev.py. We don't need excess variables(like EMAIL_FILE_PATH) in production settings, where base.py will also be imported.